src/tests/nativeaot/SmokeTests/UnitTests/Delegates.cs

```diff
@@ -51,12 +51,10 @@ public static int Run()
             result = Fail;
         }
 
-<<<<<<< HEAD
 #if !CODEGEN_WASM // TODO-LLVM: fails trying to create a thread. We need correct TryEnsureSufficientExecutionStack.
-=======
->>>>>>> runtime/main
         TestLinqExpressions.Run();
         TestDefaultInterfaceMethods.Run();
```

Comment:
Does `TestDefaultInterfaceMethods` not work? Bit surprising.

Reply:
it is working, reinstated.